cli/tests/init_worker_workspace_and_typecheck.sh

```
@@ -0,0 +1,67 @@
+if [ -d "output/init-worker-workspace" ]; then
+    rm -rf output/init-worker-workspace
+fi
+
+mkdir -p output/init-worker-workspace
+cd output/init-worker-workspace
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)